src/componentAPI.ts

```diff
@@ -318,7 +318,10 @@ export class ComponentAPI {
             throw new Error("Component Ports of: " + threeduftypesting + " not found in library");
         }
         const definition = ComponentAPI.library[threeduftypesting].object;
-        const params_to_pass = Object.fromEntries(params);
+        const params_to_pass: { [index: string]: any } = {};
```

Comment:
Will this break other stuff where this is being used ?

Reply:
Ought not, it's only used to set the port locations